### ISSUE TITLE
SNIProxy is not listening for any connections

### ISSUE THREAD

**Issue by alichry** [dlundquist/sniproxy#297]
I'm currently running sniproxy in a docker container. Omitting a trailing newline to the config file will cause sniproxy to not function properly. You can reproduce this issue by simply removing the trailling newline from the config file. 

#146 -- maybe related

![bug1](https://user-images.githubusercontent.com/32630359/42175769-4f232454-7e2f-11e8-8b8f-c4fc069aad54.png)
![bug2](https://user-images.githubusercontent.com/32630359/42175772-52f049b8-7e2f-11e8-88c1-838b739960e3.png)

**Comment by dlundquist** [comment]
Interesting, I've been able to reproduce this with config_test so this is an issue in the core parser module.

```
 [dustin@blacktop tests (master)]$ perl -pi -e 'chomp if eof' ../sniproxy.conf 
[dustin@blacktop tests (master)]$ git diff ../sniproxy.conf
diff --git a/sniproxy.conf b/sniproxy.conf
index ac3cfa2..6f8e202 100644
--- a/sniproxy.conf
+++ b/sniproxy.conf
@@ -151,4 +151,4 @@ table {
     # If no port is specified the port of the incoming listener is used
     example.com 192.0.2.10
     example.net 192.0.2.20
-}
+}
\ No newline at end of file
[dustin@blacktop tests (master)]$ ./config_test 
# ... output missing "default" table
```

**Comment by alicherri** [comment]
Yes, If I remember correctly if you define the default table before EOF you'll get an error that table "default" does not exist. That's why I had to define it before the listen block.

**Comment by dlundquist** [comment]
Fixed in e57bbdd.
